packages/aws-lambda/test/integration_test/test.js

```diff
@@ -813,6 +825,91 @@ function registerTests(handlerDefinitionPath) {
         ));
   });
 
+  describe('triggered by AWS SDK Lambda invoke function with parent span from an empty context', function () {
+    const control = prelude.bind(this)({
+      handlerDefinitionPath,
+      trigger: 'invoke-function',
+      instanaEndpointUrl: backendBaseUrl,
+      instanaAgentKey,
+      traceIdContext: 'test-trace-id',
+      spanIdContext: 'test-span-id',
+      traceLevelContext: '1'
+    });
+
+    it('must instrument with Instana headers coming from clientContext', () =>
+      verify(control, {
+        error: false,
+        expectMetrics: true,
+        expectSpans: true,
+        trigger: 'aws:sdk.invoke',
+        parent: {
+          t: 'test-trace-id',
+          s: 'test-span-id'
+        }
+      })
+        .then(() => control.getSpans())
+        .then(spans =>
+          expectExactlyOneMatching(spans, [
+            span => expect(span.n).to.equal('aws.lambda.entry'),
+            span => expect(span.k).to.equal(constants.ENTRY),
+            span => expect(span.data.http).to.not.exist
+          ])
+        ));
+  });
+
+  describe('triggered by AWS SDK Lambda invoke function with parent span from a non-empty context', function () {
+    const control = prelude.bind(this)({
+      handlerDefinitionPath,
+      trigger: 'invoke-function',
+      instanaEndpointUrl: backendBaseUrl,
+      instanaAgentKey,
+      traceIdContext: 'test-trace-id',
+      spanIdContext: 'test-span-id',
+      traceLevelContext: '1',
+      fillContext: true
+    });
+
+    it('must instrument with Instana headers coming from clientContext', () =>
+      verify(control, {
+        error: false,
+        expectMetrics: true,
+        expectSpans: true,
+        trigger: 'aws:sdk.invoke',
+        parent: {
+          t: 'test-trace-id',
+          s: 'test-span-id'
+        }
+      })
+        .then(() => {
+          const lambdaContext = control.getClientContext();
+          expect(lambdaContext && lambdaContext.Custom).to.exist;
+          expect(lambdaContext && lambdaContext.Custom && lambdaContext.Custom.awesome_company).to.equal('Instana');
+        })
+        .then(() => control.getSpans())
+        .then(spans =>
+          expectExactlyOneMatching(spans, [
+            span => expect(span.n).to.equal('aws.lambda.entry'),
+            span => expect(span.k).to.equal(constants.ENTRY),
+            span => expect(span.data.http).to.not.exist
+          ])
+        ));
+  });
+
+  describe('triggered by AWS SDK Lambda invoke function with parent span from an empty context', function () {
```

Comment:
The title looks copied/pasted - maybe it should mention x-instana-l or trace suppression.